### ISSUE TITLE
research about feedin tarifs for countries in Europe

### ISSUE THREAD

**Issue by Piranias** [greco-project/pvcompare#194]

**Comment by Piranias** [comment]
I have started a research, but did not find anything handy to use. The closest I got to an overview over european countries was this:
http://www.res-legal.eu/en/compare-support-schemes/

**Comment by SabineHaas** [comment]
Maybe one of the supporting students can research the values at least for a few countries that 1) have a feed-in tariff and 2) are interesting for us --> e.g. several locations for CPV?

**Comment by SabineHaas** [comment]
Answers I got from our partners in Portugal and Spain:

Spain:
"_The situation here in Spain is similar in a way to that of Portugal: there are **no new feed-in tariff assignments**, but those that were approved in the past remain valid.
The "sun-tax" was indeed dropped, and now the regulation allows a self-consumption regime in which the **excess of electricity is sold to the utility** (the price is determined by the electricity market, but it is near the pool price, i.e., around 4-5 c€/kWh). As far as I know there are no general subsidies for new installations, but it is true these kind of decisions are decentralized and some regional or local governments may implement some schemes (for example, it is typical now that local governments reduce the "building tax" if you install a PV system at your roof._"

Portugal:
"_Regarding the tariffs for photovoltaic energy, I don't have the full picture of Spain, so maybe our spanish colleagues can help 
you better. For example, for Portugal there is no longer any feed-in tariff assignment to photovoltaic energy, whether at the domestic or centralized production level. Only feed-in tariffs allocated a few years ago remain in force as long as these contracts are valid, but they will certainly disappear as well.
Indeed, it would be difficult, politically (and rationally), for the Portuguese government to justify today the allocation of feed-in tariffs to traditional photovoltaic technology, given that it is currently the cheapest technology for electricity production in Portugal (in fact, we achieved the world record in the new solar capacity auction 2020 with the price of $ 0.01316/kWh)._"
